testssl.sh

```diff
@@ -8352,10 +8352,10 @@ certificate_info() {
                                         let ret++ ;;
           esac
           out "$short_keyAlgo "
-          # https://tools.ietf.org/html/rfc4492,  http://www.keylength.com/en/compare/
-          # http://infoscience.epfl.ch/record/164526/files/NPDF-22.pdf
-          # see http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57_part1_rev3_general.pdf
-          # Table 2 @ chapter 5.6.1 (~ p64)
+          # https://tools.ietf.org/html/rfc4492,  https://www.keylength.com/en/compare/
+          # https://doi.org/10.1007/s00145-001-0009-4
+          # see http://dx.doi.org/10.6028/NIST.SP.800-57pt1r4
```

Comment:
https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-4/final

Reply:
I thought DOI will be better because it shows the current version of the document so it is not necessary to keep track of changing URL.